Banner notifications and customer mass actions (#232)
AdobeDocs/commerce-extensibility: src/pages/admin-ui-sdk/extension-points/banner-notification.md

---
title: bannerNotification
description: Create a banner notification mass action in the Adobe Commerce Admin.
keywords:
  - App Builder
  - Extensibility
---

# bannerNotification

> The `bannerNotification` extension point defines the contents of banner notifications that display when attempting a customer, order, or product mass action.

Reviewer: bannerNotification works also for order view button. I updated the wiki with an example. 
https://wiki.corp.adobe.com/display/DMSArchitecture/bannerNotification

> The `bannerNotification` extension point defines the contents of banner notifications that are displayed when a customer, order, or product mass action is attempted.

Reviewer 2: ```suggestion
The `bannerNotification` extension point defines the contents of banner notifications that display when attempting a customer, order, or product mass action.
```

            successMessage: 'Selected orders were updated successfully',
            errorMessage: 'App could not proceed with mass action on selected orders'
          }
        ],
        product: [
          {
            actionId: `${extensionId}::mass-action`,
            successMessage: 'Selected products were updated successfully',
            errorMessage: 'App could not proceed with mass action on selected products'
          }
        ],
        customer: [
          {
            actionId: `${extensionId}::mass-action`,
            successMessage: 'Selected customers were updated successfully',
            errorMessage: 'App could not proceed with mass action on selected customers'
          }
        ]
      }
    }
}
```

## Parameters

| Field | Type | Required | Description |
| --- | --- | --- | --- |
`customer.actionId` | Yes | The `actionId` of a customer mass action.
where banner notification will be customized. It should be the same one that is defined in the customer mass action extension point.
`customer.successMessage` | No | The success message to display when mass action is successful. A default message is displayed if this parameter is not defined.

> `customer.errorMessage` | No | The error message to display when mass action is successful. A default message is displayed if this parameter is not defined.

Reviewer: ```suggestion
`customer.errorMessage` | No | The error message to display when mass action fails. A default message is displayed if this parameter is not defined.
```

`order.actionId` | string | Yes | The `actionId` of an [order mass action](./order/mass-action.md).
`order.successMessage` | No | The success message to display when mass action is successful. A default message is displayed if this parameter is not defined.

> `order.errorMessage` | No | The error message to display when mass action is successful. A default message is displayed if this parameter is not defined.

Reviewer: ```suggestion
`order.errorMessage` | No | The error message to display when mass action fails. A default message is displayed if this parameter is not defined.
```

`product.actionId` | Yes | The `actionId` of a [product mass action](./product/mass-action.md).
`product.successMessage` | No | The success message to display when mass action is successful. A default message is displayed if this parameter is not defined.

> `product.errorMessage` | No | The error message to display when mass action is successful. A default message is displayed if this parameter is not defined.

Reviewer: ```suggestion
`product.errorMessage` | No | The error message to display when mass action fails. A default message is displayed if this parameter is not defined.
```
